src/main/java/fi/aalto/cs/apluscourses/model/Exercise.java

```diff
@@ -28,30 +28,34 @@
 
   private final boolean submittable;
 
+  private final double latePenalty;
+
   /**
    * Construct an exercise instance with the given parameters.
-   *
-   * @param id                The ID of the exercise.
+   *  @param id                The ID of the exercise.
    * @param name              The name of the exercise.
    * @param htmlUrl           A URL to the HTML page of the exercise.
    * @param userPoints        The best points that the user has gotten from this exercise.
    * @param maxPoints         The maximum points available from this exercise.
    * @param maxSubmissions    The maximum number of submissions allowed for this exercise.
+   * @param latePenalty       The penalty applied if the best submission is late.
```

Comment:
%, right?

Reply:
it's from 0.0 to 1.0, so is it called a multiplier?